torch_xla/csrc/runtime/distributed_runtime.h

```diff
@@ -3,33 +3,78 @@
 
 #include <memory>
 
+#include "torch_xla/csrc/runtime/debug_macros.h"
+#include "tsl/distributed_runtime/preemption/preemption_sync_manager.h"
 #include "xla/pjrt/distributed/distributed.h"
 
 namespace torch_xla {
 namespace runtime {
 
+// DistributedRuntime serves as the point of entry for all operations which
+// required the XLA distributed runtime, such as preemption coordination.
 class DistributedRuntime {
```

Comment:
I really dislike the naming choice for the upstream `xla::DistributedRuntime`, since it's not actually a distributed runtime. Since this class is becoming more than just a wrapper around `xla::DistributedRuntimeService` and `xla::DistributedRuntimeClient`, what do you think of changing the name to something more intuitive? e.g. `XlaCoordinator`

Reply:
Totally agree, XlaCoordinator it is! I'll update the pybinds as well.

Reply:
I left as-is for now to keep this change minimal, we can revisit in the upcoming refactor.